sqlglot/dialects/databricks.py

```diff
@@ -65,6 +84,9 @@ class Generator(Spark.Generator):
                     transforms.unnest_to_explode,
                 ]
             ),
+            exp.JSONExtract: _jsonextract_sql,
+            exp.JSONExtractScalar: _jsonextract_sql,
```

Comment:
We don't need to define `_jsonextract_sql`, consider this instead:

```python
            exp.JSONExtract: lambda self, e: binary(e, ":"),
            exp.JSONExtractScalar: lambda self, e: binary(e, ":"),
```

Reply:
Looks like `binary` introduces whitespace between the operands / path parts, if it's useful I can refactor it to be more modular in that regard otherwise will keep `_jsonextract_sql`